packages/editor-ui/src/mixins/pushConnection.ts

```diff
@@ -291,6 +291,30 @@ export const pushConnection = defineComponent({
 				}
 			}
 
+			if (receivedData.type === 'workflowFailedToActivate') {
+				this.workflowsStore.setWorkflowInactive(receivedData.data.workflowId);
+				this.workflowsStore.setActive(false);
+
+				this.showError(
+					new Error(receivedData.data.errorMessage),
+					this.$locale.baseText('workflowActivator.showError.title', {
+						interpolate: { newStateName: 'deactivated' },
```

Comment:
```suggestion
						interpolate: { newStateName: 'activated' },
```

The modal title is currently shown as `Workflow could not be deactivated:` which seems a bit misleading.

Reply:
Please see [a3147a9](https://github.com/n8n-io/n8n/pull/7643/commits/a3147a926a9501bf373b0bf5a63717f17fca97e8)